src/Component/src/Symfony/ExpressionLanguage/Provider/NotFoundOnNullExpressionFunctionProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Symfony\ExpressionLanguage\Provider;
+
+use Symfony\Component\ExpressionLanguage\ExpressionFunction;
+use Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+final class NotFoundOnNullExpressionFunctionProvider implements ExpressionFunctionProviderInterface
+{
+    private const DEFAULT_MESSAGE = 'Requested page is invalid.';
```

Comment:
Does it need anything beyond `Not found.`?
But at any case, `invalid` shouldn't be used with not found code.

```suggestion
    private const DEFAULT_MESSAGE = 'Not found.';
```

Reply:
yep, that was the message on the previous service, but this one is better, thx.